lib/deep-equal.test.js

```diff
@@ -556,27 +567,54 @@ describe("deepEqual", function() {
         assert.isFalse(samsam.deepEqual(obj, instance));
     });
 
-    it("fails different DOM elements", function() {
-        var element = document.createElement("div");
-        var el = document.createElement("div");
+    it("fails unequal errors", function() {
+        var error1 = new Error();
+        var error2 = new Error();
 
-        assert.isFalse(samsam.deepEqual(element, el));
+        assert.isFalse(samsam.deepEqual(error1, error2));
     });
 
-    it("does not modify DOM elements when comparing them", function() {
-        var el = document.createElement("div");
-        document.body.appendChild(el);
-        samsam.deepEqual(el, {});
+    describe("DOM", function() {
+        before(function() {
+            if (typeof document !== "undefined") {
+                this.samsam = samsam;
```

Comment:
Maybe you want to `delete this.samsam` in the `after` block, so that it is `undefined` for any test suites that follow this one.

Reply:
Good idea 👍